regression/cbmc-java/LocalVarTable4/live_range_exception.java

```diff
@@ -0,0 +1,20 @@
+
+public class live_range_exception {
+
+    public static void main() {
+
+	int x;
+	int y;
+	try {
+	    x = 0;
+	    y = 0;
+	}
+	catch(Exception e) {
+	    x = 1;
+	    y = 1;
+	}
+	assert(x == 0 || x == 1);
+
+    }
+
+}
```

Comment:
Elsewhere it was suggested that the coding guidelines should also be applied in regression tests. Hence I'm herewith echoing this.